pkg/cliplugins/bind/plugin/bind.go

```diff
@@ -0,0 +1,176 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugin
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/kcp-dev/logicalcluster"
+	"github.com/spf13/cobra"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/client-go/rest"
+
+	apisv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	kcpclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
+	"github.com/kcp-dev/kcp/pkg/cliplugins/base"
+	pluginhelpers "github.com/kcp-dev/kcp/pkg/cliplugins/helpers"
+)
+
+// BindOptions contains the options for creating an APIBinding.
+type BindOptions struct {
+	*base.Options
+	// WorkspacePath refers to the absolute reference to the APIExport workspace
+	// to which the user wants to bind.
+	workspacePath logicalcluster.Name
+	// Name of the APIExport.
+	apiexportName string
```

Comment:
It doesn't look like this should be here - it is calculated in `Run()` and not used outside of there.